### DISCUSSION TITLE
How do I make my models anonymous

### DISCUSSION THREAD

**Discussion by Jimbo994** [Lightning-AI/pytorch-lightning#15191] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Hi, 

I am submitting my code and trained models as supplementary information to a journal that uses double-blind reviewing.
When searching for my name, I found that my models (.ckpt files) contain paths that contain my name several times...
See an example below:

![Screenshot 2022-10-19 at 14 38 18](https://user-images.githubusercontent.com/23170678/196693283-57c0953a-3a51-4b26-97ad-4654fefea805.png)

Just to be sure, I like these paths to be anonymized by replacing anything related to my name by xxx.
I tried doing this using the sed Linux command, but unfortunately, this corrupts the files and PyTorch Lightning cannot open them anymore. 

I also tried some load and save methods from Pytorch with no success. Does anyone know how to fix this?

Kind regards,
JB

**Comment by akihironitta** [comment]
I don't think modifying binary files directly works as you've seen already. Instead, loading your ckpt file with `torch.load("asdf.ckpt")` and replacing all occurrences that you want to hide should work.

**Reply by akihironitta** [reply]
Or maybe run your script within a container :)
